personal_assistant/main.py

```diff
@@ -78,6 +78,38 @@ def main():
         elif command == "entries":
             print(num_records(personal_assistant))
 
+        elif command == "add-note":
```

Comment:
![image](https://github.com/nazarsmith/ppap7856-pythoneers-brigade-prjct/assets/61598939/f777e463-79da-4e7c-901b-1ea0839dd8ab)
I think some basic default values need to be added to avoid cases with empty name, tags, and text :)